src/api/products/products.service.js

```diff
@@ -10,7 +13,37 @@ async function getById({ id }) {
   return product;
 }
 
+async function getRecommended({ userId }) {
+  const userOrders = await ordersService.getOrdersByUserId({ userId });
```

Comment:
Is it necessary to enter Orders at Service level?

Reply:
You mean we could go directly to repo layer?
Jona adressed this the other day. If I understood it correctly, it is preferable to go to the service layer, and not directly to repo.

Reply:
That's it, usually it is a better approach to connect functionality between repositories/models at the service level. Services act like workers for their models. In this way, the code becomes easier to maintain as the solutions grows